src/nls/root/strings.js

```diff
@@ -582,6 +583,14 @@ define({
     "CMD_RESTART_NODE"                          : "Restart Node",
     "CMD_SHOW_ERRORS_IN_STATUS_BAR"             : "Show Errors in Status Bar",
     "CMD_OPEN_BRACKETS_SOURCE"                  : "Open {APP_NAME} Source",
+    "ERROR_CREATING_LAUNCH_SCRIPT"              : "An erorr occured while creating {APP_NAME} launcher script at /usr/local/bin. Please refer to https://github.com/adobe/brackets/wiki/Command-Line-Arguments for troubleshooting. Reason: ",
```

Comment:
What about:

`An error occurred while creating the {APP_NAME} command line tool at /usr/local/bin. Please refer to https://github.com/adobe/brackets/wiki/Command-Line-Arguments for troubleshooting. Reason: `